ucrop/src/main/java/com/yalantis/ucrop/UCropActivity.java

```diff
@@ -566,7 +566,10 @@ private void setAngleTextColor(int textColor) {
 
     private void setScaleText(float scale) {
         if (mTextViewScalePercent != null) {
-            mTextViewScalePercent.setText(String.format(Locale.getDefault(), "%d%%", (int) (scale * 100)));
```

Comment:
@ashiagr , I would suggest using 

                String.format(Locale.getDefault(), "%d%%", Math.round(scale/ initialScale * 100))

This will prevent incorrect value like 99.99% when it should be 100%. Also integer value should be enough to display. Fractional part is redundant.